src/terminal/Process.cpp

```diff
@@ -472,8 +477,24 @@ string Process::workingDirectory() const
 		// ignore failure, and use default instead.
 		return "."s;
 	}
+#elif defined(__APPLE__)
+    try {
+        struct proc_vnodepathinfo vpi;
```

Comment:
```suggestion
        auto  vpi = proc_vnodepathinfo{};
```
I don't mind not using the auto-way here, but I'd prefer to have at least the `struct` keyword removed. using `{}` in the end ensures initialization of all fields to their default values.